src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -150,6 +150,21 @@
   end
 
   describe 'GET #users' do
+    before do
+      @project = create(:project)
+      create(:relationship_project_user, project: @project, user: create(:confirmed_user))
+      create(:relationship_project_user, project: @project, user: create(:confirmed_user))
```

Comment:
Yes, I wanted to have more than one relationship.. but I will change it to use **create_list** instead